draft-dnsop-deleg-transport.md

```diff
@@ -208,31 +217,33 @@ alpn=doq tlsa=disabled mandatory=alpn
 ~~~
 {: title="tlsa SvcParam Examples for DELEG"}
 
-## Deployment Considerations
+## Resolution procedure
 
-The DELEG and SVCB records intends to replace the NS record while also adding additional functionality in order to support additional transports for the DNS. Below are discussions of considerations for deployment.
+As specified in {{?I-D.draft-dnsop-deleg}}, the ServiceMode DELEG records provide a weak signal from the operator about which transport mechanisms they would most prefer clients to use, but it is not mandatory that resolvers try the delegated servers in priority order.  It might not even be possible for a resolver to do, such as if the highest priority record specifies an alpn that the resolver does not implement, but a lower priority record has an alpn that it can use.
```

Comment:
I am still not sure if we actually want priority.

Reply:
I'm ambivalent on it.   While we still would need a way to distinguish AliasMode from ServiceMode, I'm kind of neutral about using a special magic 0 priority to do so.  I can see some limited use for it, but I also think it might be an attractive nuisance.  "Oh," says the Black hat,  "so the put this service at a lower priority?  Let's make that our  target!"